### PR TITLE
Video page

### PR DESCRIPTION
**Pull request by vishalg8454**
This PR adds VideoPage for individual videos. Video can be played from the video page

The pages added are - 
- `VideoPage` at "src/pages/video-page"

[live preview](https://deploy-preview-5--videostock.netlify.app/)

Would love to hear your feedback and suggestions.

image

### CODE REVIEW

File: src/pages/video-page/VideoPage.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+const VideoPage = () => {
+  let { videoId } = useParams();
+  const [videoData, setVideoData] = useState({});
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const res = await axios.get(`/api/video/${videoId}`);
+        setVideoData(res.data.video);
+        console.log(res.data.video);
```
Comment (arpit01923) [inline src/pages/video-page/VideoPage.jsx:19]: remove unnecessary console

**Review by arpit01923 — APPROVED** [review]
- clean code 
- functionality works fine
- good to merge

File: src/pages/homepage/Homepage.jsx [hunk 2]
```diff
@@ -53,15 +54,20 @@ const Homepage = () => {
...
-              channelUrl={channelUrl}
-              videoTitle={videoTitle}
-              channelName={channelName}
-              viewCount={viewCount}
-              publishedDate={publishedDate}
+            <Link
               key={_id}
-            />
+              to={`video/${_id}`}
+              style={{ textDecoration: "none", color: "black" }}
```
Comment (syed-shaida-hussain) [inline src/pages/homepage/Homepage.jsx:60]: Try to avoid inline styling

**Review by syed-shaida-hussain — APPROVED** [review]
Try avoiding inline styling and remove console logs.

File: src/pages/homepage/Homepage.jsx (hunk 2 above)
Comment (vishalg8454) [reply src/pages/homepage/Homepage.jsx:60]: sure.

**Pull request merged**